kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -44,6 +49,6 @@
     {{/each}}
     </tbody>
 </table>
-<div class='kb-data-staging-footer'>
+<div class='kb-staging-table-footer'>
```

Comment:
should this have a different name so it isn't confused with a table foot (`tfoot`)?